nibabel/cifti2/tests/test_new_cifti2.py

```diff
@@ -367,6 +384,7 @@ def test_plabel():
     with InTemporaryDirectory():
         ci.save(img, 'test.plabel.nii')
         img2 = ci.load('test.plabel.nii')
```

Comment:
is this a valid CIFTI type? I cannot find *plabel.nii in the specifications
https://www.nitrc.org/forum/attachment.php?attachid=341&group_id=454&forum_id=1955

Reply:
there is a plabel: https://github.com/Washington-University/workbench/blob/master/src/Commands/CommandParser.cxx#L661

but indeed i don't see it in the spec.

Reply:
also here: https://github.com/Washington-University/workbench/blob/6c7414d9ba379604c10673d6d68eeeb68f1332e9/src/Common/DataFileTypeEnum.cxx#L207

Reply:
If we will raise an exception upon saving, we have some options

* remove the plabel test
* alter `ci.save`'s signature to include a new parameter, `force=False`, that can be switched on to allow saving without intent codes

WDYT?

Reply:
how about skipping the plabel test for now till we find the INTENT code. i would rather not have a parameter.

Reply:
@effigies i had no luck finding it as well
https://github.com/Washington-University/workbench/blob/1b79e569e8b8277f4abf71fabd4afc2889849362/src/FilesBase/nifti2.h#L32-L49

Reply:
plabel is not in the cifti-2 spec.  We didn't see it being useful at the time, so it was not in the draft.  We later found that the information needed to reorder parcellated files to group certain parcels together in matrix display could be stored as a parcels by labels file, and decided to use that extention, matching the previous pattern of extension naming.

Per the "unknown" special cifti intent (main cifti-2 document, top of page 13, middle of paragraph), it is in fact valid to make such a file, and its extension is technically open to be anything that ends in ".<something>.nii".  Feel free to write a test that loads and/or saves a cifti mapping combination that is not in the spec, this is an expected use case.